pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -60,6 +62,14 @@ func LoadClientCert(kubeconfigPath string, bootstrapPath string, certDir string,
 	if err != nil {
 		return fmt.Errorf("unable to load bootstrap kubeconfig: %v", err)
 	}
+	client, err := kubernetes.NewForConfig(bootstrapClientConfig)
+	if err != nil {
+		return err
+	}
+	_, err = client.ServerVersion()
```

Comment:
I'm not sure what value this provides over simply returning the error we get when we try to request the CSR in `RequestNodeCertificate`